mix.exs

```diff
@@ -27,7 +27,7 @@ defmodule Sentry.Mixfile do
       {:hackney, "~> 1.8 or 1.6.5"},
       {:uuid, "~> 1.0"},
       {:poison, "~> 1.5 or ~> 2.0 or ~> 3.0"},
-      {:plug, "~> 1.0", optional: true},
+      {:plug, ">= 1.6.0 and < 2.0.0", optional: true},
```

Comment:
Could you change this to `~> 1.6`? I think that would be equivalent to the above?

Reply:
All good, thanks for addressing git!

If it's helpful, https://hexdocs.pm/elixir/Version.html has an explanation of the version locking rules